ab_tool/views/experiment_pages.py

```diff
@@ -96,11 +96,16 @@ def submit_edit_experiment(request, experiment_id):
     name = experiment_dict["name"]
     notes = experiment_dict["notes"]
     if experiment.tracks_finalized:
-        # Only allow updating name and notes for started experiments
+        # Only allow updating name, notes, and track names for started experiments
         experiment.update(name=name, notes=notes)
+        existing_tracks = [i for i in experiment_dict["tracks"] if i["id"] is not None]
+        for track_dict in existing_tracks:
+            track = Track.get_or_404_check_course(track_dict["id"], course_id)
```

Comment:
I think here I'd compare the json track's `originalName` with the track's `name` field before I would retrieve and ultimately update the track.  I think it would potentially save a number of db calls.